code/modules/paperwork/paper.dm

```diff
@@ -867,3 +867,9 @@
 		to_chat(user, span_warning("You arm the incineration mechanism."))
 		armed = TRUE
 	return CLICK_ACTION_SUCCESS
+
+/obj/item/paper/selfdestruct/job_application
```

Comment:
Woah wtf you can't add this??????

Reply:
nt reps deserve a spell that conjures job applications for the funny